### PR TITLE
Moving dev to 6.x

### PR DESCRIPTION
**Pull request by rakatyal**

### CODE REVIEW

File: www/docs/de/6.x/guide/platforms/blackberry10/home.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+title: Blackberry 10 Guides
+---
+
+# Blackberry 10 Guides
+
+* [BlackBerry 10 Platform Guide](index.html)
+* [BlackBerry 10 Shell Tool Guide](tools.html)
+* [BlackBerry 10 Configuration](config.html)
+* [BlackBerry 10 Plugins](plugin.html)
+* [Upgrading BlackBerry 10](upgrade.html)
```
Comment (dblotsky) [inline www/docs/de/6.x/guide/platforms/blackberry10/home.md]: Why is there a new page in German?

**Comment by dblotsky** [comment]
@rakatyal it looks like some pages were added to other languages than English. Was this intentional?

**Comment by rakatyal** [comment]
Yes. They were added as part of [this](https://github.com/apache/cordova-docs/pull/507/files) commit. We added landing pages for guides which we didn't make single page.

**Comment by dblotsky** [comment]
@rakatyal if they were part of a previous commit then please rebase this branch on top of master so that they don't show up in this PR.

**Comment by rakatyal** [comment]
They were added to dev and now they are being added to 6.x

**Comment by dblotsky** [comment]
Ohh, I see. That makes sense now. Thanks for explaining. So then this change is technically for all languages, yes?

**Comment by rakatyal** [comment]
Yup this will make them access the platform guides for other languages as well.

**Comment by dblotsky** [comment]
Cool. LGTM.

**Pull request closed without merging**